users/updates/components/VLBp32gTWvEo.php

```diff
@@ -7,7 +7,7 @@
 
 $countE=0;
 
-$db->query("ALTER TABLE users ADD COLUMN vericode_expiry timestamp AFTER `vericode`");
+$db->query("ALTER TABLE users ADD COLUMN vericode_expiry timestamp AFTER vericode");
```

Comment:
Is there an `AFTER` keyword in PostgreSQL?

Reply:
So I was confused with this AI suggested there is no AFTER keyword one time. I even googled it, but I am unsure if this keyword can be used. I found an example of it being used with a TRIGGER. Also, the keyword shows here on the Postgre website.

https://www.postgresql.org/docs/current/sql-keywords-appendix.html

It shows as non-reserved, which means it has context in some areas but not all. What do you this, professor? I will test it, too and let you know,

Reply:
Ok, so it already exists in the sql.sql file. I don't know how else to test it. Shouldn't we take dump of the database before the installation, like when the updates are not applied, so that these updates can be applied later when the user goes to /users/updates or click the link on the dashboard?

Reply:
The `AFTER` keyword is used in PostgreSQL's [trigger system](https://www.postgresql.org/docs/current/plpgsql-trigger.html). It cannot be used for specifying the position of a newly inserted column. Please remove and check for other occurrences.